lib/src/models/message.dart

```diff
@@ -0,0 +1,23 @@
+class Message {
+  /// Whether the message was sent by the Explorer.
+  final bool isLocal;
```

Comment:
I'm not a huge fan of this name. I toyed with names like `sentByExplorer`, `fromExplorer` and `fromMe`, but none of them felt great. For now, since we're using "local" and "remote" when referring to the Explorer and Agent streams, we'll use the same terms here.